merchants/verkkokauppa/helpers.py

```diff
@@ -77,9 +79,12 @@ def create_verkkokauppa_order(reservation: Reservation):
 def _get_order_params(reservation: Reservation):
     runit = reservation.reservation_unit.first()
     quantity = 1  # Currently, we don't support quantities larger than 1
-    price_net = quantity * reservation.price_net
-    price_vat = (
-        quantity * reservation.price_net * (reservation.tax_percentage_value / 100)
+    price_net = round_decimal(Decimal(quantity * reservation.price_net), 2)
```

Comment:
The `reservation.price_net` should be the total price without taxes. Meaning multiplying with quantity here isn't necessary 🤔